### ISSUE TITLE
error when use docker 

### ISSUE THREAD

**Issue by luzhang92** [sdnfv/openNetVM#36]
when I run the opennetvm's example in docker, it occurs the following problem: /openNetVM/examples/basic_monitor/build/app/monitor: error while loading shared libraries: libnuma.so.1: cannot open shared object file: No such file or directory

**Comment by nks5295** [comment]
I'm facing the same problem on my setup.  Thanks for the catch - let me debug and see what I can come up with.

**Comment by nks5295** [comment]
The problem is that by default, the container deployed does not have libnuma-dev installed on it.  We updated DPDK version a while back and that introduced this dependency.  I will submit a fix for this shortly!

**Comment by nks5295** [comment]
Sorry for the late response -- but I have submitted #38 which is under review.  This will resolve your issues.

To summarize, the base Ubuntu 14.04 image did not have the dependencies needed to run the latest version of DPDK.  Our Docker script used to run containers under the Docker network "none".  That prevented users from running commands like `apt install ...` or anything that required a network.  The PR I posted changes the Docker script to use our [OpenNetVM Docker image](https://hub.docker.com/r/sdnfv/opennetvm/), which has the relevant dependencies pre-installed on an Ubuntu 14.04 base.  The PR also updates the Docker script to launch containers on the "bridge" Docker network.

Hope that helps!
